test/html-comment-in-markdown-table.md

```diff
@@ -51,3 +51,7 @@
 | cell  | <!--           \
 | cell  | `{MD038} ` --> |
 | cell  | cell           |
+
+<!-- markdownlint-configure-file {
+  "MD055": false
```

Comment:
Good. As of a couple of weeks ago, the standard is to use the rule name instead of its number when configuring like this.